src/reducers/root-reducer.js

```diff
@@ -0,0 +1,4 @@
+var changeFixture = require('./change-fixture.js');
+
+// only exports one reducer now, please use combineReducers when more are added
+module.exports = changeFixture;
```

Comment:
why not usign `combineReducers({changeFixture: changeFixture})` for a single reducer?